Configurations/unix-Makefile.tmpl

```diff
@@ -654,9 +654,11 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/ck_errf.pl -strict */*.c */*/*.c )
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl -internal )
 	( cd $(SRCDIR)/engines; \
-	  for e in *.ec; do \
-	      $(PERL) ../util/mkerr.pl -conf $$e -rebuild *.c; \
-	  done )
+          $(PERL) ../util/mkerr.pl -conf e_capi.ec e_capi.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_dasync.ec e_dasync.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_ossltest.ec e_ossltest.c ; \
+          cd afalg; \
+          $(PERL) ../../util/mkerr.pl -conf e_afalg.ec e_afalg.c )
```

Comment:
yeah it did work, sort of by accident; because of the "*.c" on every line.  Yuk.
I will fix this if/when it becomes the only issue gating approval of this PR.